### ISSUE TITLE
Is there any way to keep fluent bit crd modify after kubesphere ClusterConfiguration update?

### ISSUE THREAD

**Issue by LQBing** [fluent/fluentbit-operator#162]
I installed fluentbit with enable kubesphere logging plugin.
My cluster cri is containerd, not docker. So I need change input CRDs.
But if kubesphere ClusterConfiguration update, the logging plugin will be reinstalled, those CRDs I changed will be overwrite by ks-installer.
Any solution?

**Comment by benjaminhuo** [comment]
@wenchajun Do we still have this issue in v3.2?

**Comment by wenchajun** [comment]
What is your version of Kubesphere? An adaptation for containerd is available in kubespherev3.2.Each change in ClusterConfiguration will cause the ks-Installer to be reinstalled, but the original configuration will be skipped.

**Comment by LQBing** [comment]
> What is your version of Kubesphere? An adaptation for containerd is available in kubespherev3.2.Each change in ClusterConfiguration will cause the ks-Installer to be reinstalled, but the original configuration will be skipped. This is the automatic installation of Ks-Installer, maybe you can go to Ks-Installer to raise the relevant issue

Kubesphere v3.1.1

v3.2 is available? I tried install it with kubekey in my testing machine, but error info tell me there is not docker imagekubesphere/ks-installer with tag v3.2.0

**Comment by wenchajun** [comment]
> > What is your version of Kubesphere? An adaptation for containerd is available in kubespherev3.2.Each change in ClusterConfiguration will cause the ks-Installer to be reinstalled, but the original configuration will be skipped. This is the automatic installation of Ks-Installer, maybe you can go to Ks-Installer to raise the relevant issue
> 
> Kubesphere v3.1.1
> 
> v3.2 is available? I tried install it with kubekey in my testing machine, but error info tell me there is no docker imagekubesphere/ks-installer with tag v3.2.0

Yes, And kubespherev3.2  is coming soon. If you are finished installing the component, you can scale ks-Installer to 0.

**Comment by joshuabaird** [comment]
This issue is being closed due to inactivity.  Please re-open a new issue if it is still relevant.
